sgl-pdlb/src/server.rs

```diff
@@ -0,0 +1,341 @@
+use crate::io_struct::{Bootstrap, ChatReqInput, GenerateReqInput};
+use crate::strategy_lb::{EngineInfo, EngineLoad, EngineType, StrategyLB};
+use actix_web::{HttpRequest, HttpResponse, HttpServer, get, post, web};
+use bytes::Bytes;
+use futures::{Stream, StreamExt, future::join_all};
+use reqwest::StatusCode;
+use serde_json::json;
+use std::{io::Write, pin::Pin};
+
+pub enum ProxyResponseType {
```

Comment:
nit
```suggestion
pub enum ProxyResponseBody {
```

since it is not a type but has data